contracts/utils/AddressManager/AddressManager.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+// external
+import "@thales-dao/contracts/contracts/AddressManager/AddressManager.sol";
+
+contract AddressManagerExtension is AddressManager {
```

Comment:
I thought we are using the existing AddressManager?

Reply:
done